src/components/ResultsMap.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<GmapMap
+		:center="centerMap"
+		:zoom="10"
+		map-type-id="terrain"
+		style="width: 100%; height: 50vh"
+	>
+		<GmapMarker
+			v-for="(m, index) in markers"
```

Comment:
I love how `v-for` allows you to loop through the `markers` array and produce a new `GmapMarker`.

Reply:
Agreed! It's also great that we can pass down the markers as props and still loop through them so elegantly.